### DISCUSSION TITLE
A little bit lost with with model 'TS0601' and manufacturer name '_TZE200_a8sdabtg'

### DISCUSSION THREAD

**Discussion by lorddoener** [Koenkk/zigbee2mqtt#15200] [Devices]
Hello all,

I bought a temperature and humidity sensor from TuYa. I just wanted to read up on how to add currently unsupported TuYa devices to Zigbee2Mqtt.

`warning 2022-11-26 16:49:26Received message from unsupported device with Zigbee model 'TS0601' and manufacturer name '_TZE200_a8sdabtg`
`warning 2022-11-26 16:49:26Please see: https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html`

I came across the following entry in the repository:
https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/devices/tuya.js

There it looks to me like the device is already supported:

```{
        fingerprint: tuya.fingerprint('TS0601', ['_TZE200_qoy0ekbd', '_TZE200_znbl8dj5', '_TZE200_a8sdabtg']),
        model: 'ZG-227ZL',
        vendor: 'TuYa',
        description: 'Temperature & humidity LCD sensor',
        fromZigbee: [tuya.fzDataPoints],
        toZigbee: [tuya.tzDataPoints],
        configure: tuya.configureMagicPacket,
        exposes: [e.temperature(), e.humidity(), tuya.exposes.temperatureUnit(), tuya.exposes.temperatureCalibration(),
            tuya.exposes.humidityCalibration(), e.battery()],
        meta: {
            tuyaDatapoints: [
                [1, 'temperature', tuya.valueConverter.divideBy10],
                [2, 'humidity', tuya.valueConverter.raw],
                [4, 'battery', tuya.valueConverter.raw],
                [9, 'temperature_unit', tuya.valueConverter.temperatureUnit],
                [23, 'temperature_calibration', tuya.valueConverter.divideBy10],
                [24, 'humidity_calibration', tuya.valueConverter.raw],
            ],
        },
```

Where is my thinking error here? Before I read up on the subject, I just wanted to make sure I'm not reinventing the wheel.

EDIT:

Thanks for your support

**Comment by Koenkk** [comment]
Probably not supported in the release you are running, it will be supported in the next release on 1 December.

**Reply by lorddoener** [reply]
So simple 🙈. Thanks for your  great work!
